shared/src/userData.ts

```diff
@@ -46,7 +47,7 @@ export const createEmptyBusiness = (id?: string): Business => {
       roadmapOpenSteps: [],
       hiddenCertificationIds: [],
       hiddenFundingIds: [],
-      visibleSidebarCards: [],
+      visibleSidebarCards: [SIDEBAR_CARDS.notRegistered, SIDEBAR_CARDS.notRegisteredExistingAccount],
```

Comment:
pebble - So isn't this the function used to seed all new users? And doesn't Oscar not want these cards?

I could be mistaken about how this works

Reply:
The conditional rendering logic lives in `dashboard.tsx`.

Reply:
[pebble] I see that the logic in `dashboard.tsx` is overwriting `visibleSidebarCards` to whichever sidebar card needs to be present based on persona and operatingPhase. What is the value in empty userData containing these values when they are being overwritten? Additionally, by including both of these values, we are for a brief second creating a state within userData that should be impossible. A user shouldn't have values that are for both starting/foreign and existing personas. Let me know if this makes sense or if I'm misunderstanding the intention of it.

Reply:
I removed these cards from the array, per Soma's feedback.